.editorconfig

```diff
@@ -226,6 +245,11 @@ csharp_preserve_single_line_statements = true
 
 # Naming rules
 
+dotnet_naming_rule.constant_fields_should_be_pascal_case.severity = warning
+dotnet_naming_rule.private_constants_rule.severity = warning
+dotnet_naming_rule.private_constants_rule.style = pascal_case
```

Comment:
I don't personally like pascal case for `private` constants but that seems like what the rest of the solution was doing and is in the .net style guidelines.